.github/workflows/tests.yml

```diff
@@ -7,24 +7,11 @@ on:
     branches: [ master ]
 
 jobs:
-  phpunit7:
-    runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        php-versions: ['7.1', '7.2', '7.3']
-    steps:
-      - uses: actions/checkout@v2
-      - uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ matrix.php-versions }}
-      - run: composer global require phpunit/phpunit ^7
-      - run: phpunit --configuration=phpunit.xml --include-path=lib/
-
   phpunit8:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        php-versions: ['7.3', '7.4', '8.0', '8.1']
+        php-versions: ['7.4', '8.0', '8.1']
```

Comment:
With this, we can just install PHPUnit using `composer install` instead of `composer global require phpunit/phpunit ^8`, since the version in the lockfile is compatible with `>= 7.3`.

Reply:
Yes. And for the record the currently locked phpunit version is `^9`.